install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml

```diff
@@ -419,8 +419,13 @@ spec:
 {{- if and (or (and (eq .Release.Namespace "kube-system") (gt .Capabilities.KubeVersion.Minor "10")) (ge .Capabilities.KubeVersion.Minor "17") (gt .Capabilities.KubeVersion.Major "1")) .Values.enableCriticalPriorityClass }}
       priorityClassName: system-node-critical
 {{- end }}
+{{- if .Values.serviceAccounts.operator.create }}
```

Comment:
Just want to echo the comment from Sebastian. I think with the combination with default values, we don't even need to check for create flag.

```
      serviceAccount: {{ .Values.serviceAccounts.cilium.name | quote }}
      serviceAccountName: {{ .Values.serviceAccounts.cilium.name | quote }}
```

Reply:
100%. Fixed now.